src/main/resources/application.yml

```diff
@@ -0,0 +1,71 @@
+info:
+  app:
+    name: ${name:hocs-case-creator}
+    fullname: HOCS Case Creator
+    version: ${version:0.0.1}
+  namespace: local
+
+spring:
+  main:
+    banner-mode: off
+
+server:
+  port: 8092
+  shutdown: graceful
+
+camel:
+  springboot:
+    main-run-controller: false
+
+aws:
+  sqs:
+    config:
+      region: eu-west-2
+    ukvi-complaint:
+      url:
+      account:
+        access-key:
+        secret-key:
+      attributes:
+        max-messages: 10
+        wait-time: 20
```

Comment:
do we want to be explicit with these? are the defaults not sufficient? we have an incredibly general purpose need.

Reply:
The `max-messages` is useful if we want to start specifying the flow of messages (plus there is no set default by the listener). The wait-time is defaulted to 20, I suppose we could set this just for local (as we don't care about long polling).

Reply:
I have removed the explicit values for the wait-time in production so we use it's defaults. I have kept the max-messages set to 10 as there is no default set within the SDK. It will also allow us to control the flow.